src/tools/cephfs/cephfs-shell

```diff
@@ -58,8 +58,8 @@ def poutput(s, end='\n'):
     shell.poutput(s, end=end)
 
 
-def perror(msg):
-    shell.perror(msg, end='\n', apply_style=True)
+def perror(msg, end='\n', apply_style=True):
```

Comment:
It's better to remove ```end='\n', apply_style=True```. As it is not compatible with older cmd2 versions.

Reply:
But that would disable us from using these arguments in future. IMO, it's better to have them at the cost of not being compatible with older version.

OR, if there's a serious need for it, we can check for `cmd2.__version__` beforehand. But it is not necessary/vital to be compatible with older version, so I suppose we can just skip that.